### PR TITLE
[SPARK-26710][TEST] Fix some errors in image test case

### PR DESCRIPTION
**Pull request by xubo245**
## What changes were proposed in this pull request?
 Fix test errors in image test case
(Please fill in changes proposed in this fix)

## How was this patch tested?
run test case 
(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
How did you run the test btw? it passes on my local (IDE) and Jenkins.

**Comment by xubo245** [comment]
@HyukjinKwon  I git clone spark and mvn clean -DskipTests install, then run the test case, it throw exception like the jira content.  My laptop is Mac. my Idea version is 2018.1.2.

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala [hunk 1]
```diff
@@ -17,9 +17,7 @@
 
 package org.apache.spark.sql.execution.command
 
-import java.io.File
```
Comment (srowen) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala]: This looks unrelated?

File: mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala [hunk 2]
```diff
@@ -17,18 +17,19 @@
...
 import org.apache.spark.SparkFunSuite
 import org.apache.spark.ml.image.ImageSchema._
 import org.apache.spark.mllib.util.MLlibTestSparkContext
 import org.apache.spark.sql.Row
-import org.apache.spark.sql.types._
 
 class ImageSchemaSuite extends SparkFunSuite with MLlibTestSparkContext {
   // Single column of images named "image"
-  private lazy val imagePath = "../data/mllib/images/origin"
+  private lazy val imagePath = new File(this.getClass.getResource("/").getPath
```
Comment (srowen) [inline mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala]: This doesn't look right; looks like you're trying to fix the test rather than you're env. The tests pass when run in my IDE and pass when run in the build. I'd close this.

Comment (xubo245) [reply mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala]: The ubuntu and Mac  system both have this problem.  Are there any configuration except "git clone spar, mvn clean -DskipTests install, then run the test case"?

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala (hunk 1 above)
Comment (xubo245) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala]: just unused import, I think no need raise a new PR for unused import.

File: mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala (hunk 2 above)
Comment (HyukjinKwon) [reply mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala]: I use Mac but looks no problem in my IDE. Jenkins looks no problem too. Can you investigate why it is?

Comment (xubo245) [reply mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala]: ok @HyukjinKwon
